zbackup_base.hh

```diff
@@ -34,8 +34,8 @@ class ZBackupBase: public Paths
 public:
   DEF_EX( Ex, "ZBackup exception", std::exception )
   DEF_EX_STR( exWontOverwrite, "Won't overwrite existing file", Ex )
-  DEF_EX( exStdinError, "Error reading from standard input", Ex )
-  DEF_EX( exWontReadFromTerminal, "Won't read data from a terminal", exStdinError )
+  DEF_EX_STR( exInputError, "Error reading from input:", Ex )
+  DEF_EX( exWontReadFromTerminal, "Won't read data from a terminal", Ex )
```

Comment:
Sorry, I should have expand my question.

I think that in this case is better to [add](https://gist.github.com/Am1GO/e44cf2af69df19f7d029) default constructor to DEF_EX_STR and keep exParent for exWontReadFromTerminal.